src/main/java/at/ac/tuwien/kr/alpha/solver/NoGoodStoreAlphaRoaming.java

```diff
@@ -171,7 +174,7 @@ private ConflictCause addAndWatch(final NoGood noGood) {
 				}
 			}
 			// Check strong unassigned for nogoods with head.
-			if (noGood.hasHead() && noGood.getHead() != i && (literalEntry == null || literalEntry.getTruth().isMBT())) {
+			if (noGood.hasHead() && i != 0 && (literalEntry == null || literalEntry.getTruth().isMBT())) {
```

Comment:
In this file `0` occurs several times as a magic value, I guess denoting the head. There also seems to be a `HEAD` constant. Could it be used here instead of `0`?

Reply:
Yes, `HEAD` should be used. I will add that.